person.rb

```diff
@@ -1,8 +1,12 @@
-class Person
+require_relative './nameable'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
-  def initialize(age, name = 'unknown', parrent_permission: true)
+  def initialize(age, name: 'unknown', parrent_permission: true)
+    super()
+
```

Comment:
- [ ] Please make sure that the `Person` class has a `correct_name` method implemented, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Make%20sure%20that%20this%20class%20has%20a%20method%20correct_name%20implemented.%20It%20should%20simply%20return%20the%20name%20attribute.) requirement.

Reply:
thank you. this has been resolved